### PR TITLE
Reduce logs, cover edge cases in /sys mount check

### PR DESCRIPTION
**Pull request by okartau**
Looking at logs shows few possibilities to reduce redundant or not relevant logs.
While changing /sys detection logging, I discovered this function may return OK in some edge cases where search for mount point or mount option does not match, which is not what we want, so there is one commit improving that part.

### CODE REVIEW

File: test/e2e/deploy/deploy.go [hunk 1]
```diff
@@ -420,6 +420,7 @@ func EnsureDeployment(deploymentName string) *Deployment {
 		cmd.Dir = os.Getenv("REPO_ROOT")
 		cmd.Env = os.Environ()
 		cmd.Env = append(cmd.Env,
+			"TEST_DEPLOYMENT_QUIET="+string("quiet"),
```
Comment (pohly) [inline test/e2e/deploy/deploy.go:423]: This looks odd. Why the string cast, and why not simply "TEST_DEPLOYMENT_QUIET=quiet"?

Comment (okartau) [reply test/e2e/deploy/deploy.go]: thanks, good point, I had over-complicated this by copying existing line. Changed it to
"TEST_DEPLOYMENT_QUIET=quiet" and pushed.
